services/s3/index.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const {S3Client, PutObjectCommand} = require('@aws-sdk/client-s3');
+const {VError} = require('verror');
+
+const isPathForced = process.env.NODE_ENV === 'development';
+const s3Cli = new S3Client({
+    region: process.env.AWS_DEFAULT_REGION, // The default value is us-east-1. this is the localstack REGION.
+    endpoint: process.env.DCS_S3_URL,
+    maxAttempts: 4,
+    forcePathStyle: isPathForced, // needed with localstack.
+    credentials: {
+        accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+        secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY
```

Comment:
Are these variables new? If so, you'll need to 
a) Include them in `kube_deploy/{env}/deploy.yml`
b) Name them more specifically, please. We have a number of AWS credentials.

Possibly one for the "laterbase" but we now have DCS accessing multiple AWS assets, might be worth consolidating the IAM that already exists rather than continuously adding new credentials.

Reply:
I've changed this to use the SQS credentials